pkg/rancher-components/.npmrc

```diff
@@ -0,0 +1 @@
+@rancher:registry=https://npm.pkg.github.com
```

Comment:
I think it makes sense to publish to the github repository, but we can use npm if that makes more sense.

Reply:
We will use npm